package.json

```diff
@@ -18,7 +18,7 @@
     "@guardian/atom-renderer": "1.2.2",
     "@guardian/automat-modules": "^0.3.8",
     "@guardian/braze-components": "^8.1.3",
-    "@guardian/commercial-core": "^5.4.0",
```

Comment:
Is there a reason for the deleting the caret (^) here?

Reply:
No, `yarn` did that when I asked for that specific version.

There is some value in pinning all your dependencies, which tools like Mend Renovate suggest by default, but it’s not the purpose of this PR so I’ll keep it as-is.